tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -1557,6 +1634,114 @@ describe("clipboard", () => {
     expect(getStyle(model, "C2")).toEqual({});
   });
 
+  test("copy cells with CF => remove origin CF => paste => it should paste with original CF", () => {
+    const model = new Model();
+    setCellContent(model, "A1", "1");
+    setCellContent(model, "A2", "0");
+    setCellContent(model, "A3", "1");
+    const sheetId = model.getters.getActiveSheetId();
+    const cf = createEqualCF("1", { fillColor: "#00FF00" }, "cfId");
+    model.dispatch("ADD_CONDITIONAL_FORMAT", {
+      cf,
+      sheetId,
+      ranges: toRangesData(sheetId, "A1:A3"),
+    });
+    copy(model, "A1:A3");
+    model.dispatch("REMOVE_CONDITIONAL_FORMAT", {
+      id: "cfId",
+      sheetId,
+    });
+    paste(model, "D1");
+    expect(model.getters.getConditionalFormats(model.getters.getActiveSheetId())).toMatchObject([
+      { ranges: ["D1:D3"], rule: cf.rule },
+    ]);
+    expect(getStyle(model, "D1")).toEqual({ fillColor: "#00FF00" });
+    expect(getStyle(model, "D2")).toEqual({});
+    expect(getStyle(model, "D3")).toEqual({ fillColor: "#00FF00" });
```

Comment:
The computed style is already tested elsewhere. You only care about the copy/paste behavior here, not the effect of the CF
```suggestion
```